binding/dotnet/README.md

```diff
@@ -24,6 +24,10 @@ Leopard is an on-device speech-to-text engine. Leopard is:
 
```

Comment:
Requirements should be .NET 6.0 now because you can't build and test this library without that SDK. (I think we missed this in Porcupine too)